pkgs/test_api/lib/src/frontend/future_matchers.dart

```diff
@@ -66,7 +66,7 @@ class _Completes extends AsyncMatcher {
 
       var buffer = StringBuffer();
       buffer.writeln(indent(prettyPrint(value), first: 'emitted '));
-      if (result!.isNotEmpty) buffer.writeln(indent(result, first: '  which '));
+      if (result.isNotEmpty) buffer.writeln(indent(result, first: '  which '));
```

Comment:
Do CFE and analyzer match with their flow analysis here?

Reply:
yup! w/ 2.9